packages/react/src/components/input/input.tsx

```diff
@@ -1,21 +1,39 @@
 import { c, classy, InputProps as BaseProps, m } from '@onfido/castor';
 import React from 'react';
+import { FieldLabelWrapper } from '../../internal';
 import { withRef } from '../../utils';
 
+const idPrefix = 'castor_input';
+let idCount = 0;
+
 export const Input = withRef(
   (
-    { type = 'text', invalid, className, ...restProps }: InputProps,
+    {
+      id = `${idPrefix}_${idCount++}`,
```

Comment:
🤔    interesting decision not to go with uuidv4, mind sharing the thought process?

I guess this one prevents collisions 100%, rather than 1-in-a-million, but it's not opaque (which can be both an advantage and disadvantage).

Reply:
Regarding uuidv4 - as we do not use any libs, obvious was to have own utility for such. I was looking at best examples and still they seem too complex.

Then again I was thinking, does UUID best fits in such case? It's too universal, so having a custom ID system with "castor" prefix IMO is even nicer to look up for something in the DOM.

I did use such technique (if you can call it) previously with Redux, and it was actually working well.

Reply:
Fair enough. A custom uuidv4 function is simple to execute but hard to read.
I suppose the advantages of a non-opaque id outweigh the disadvantages in this case.

Should we make it start from 1 rather than 0 though?
```suggestion
      id = `${idPrefix}_${++idCount}`,
```